.github/workflows/tests-on-pr-push-and-merge.yml

```diff
@@ -3,10 +3,8 @@ on:
   workflow_dispatch: { }
   push:
     branches: [ main ]
-    paths-ignore: [ '**.md' ]
   pull_request:
     branches: [ '**' ]
-    paths-ignore: [ '**.md' ]
```

Comment:
Thanks for fixing!

FYI technically you only need to remove for `pull_request`, not `push`. See https://github.com/DataBiosphere/terra-workspace-manager/blob/main/.github/workflows/test.yml#L11

Let's future-proof this. Add comment:

```
# Branch settings require status checks before merging, so don't add paths-ignore.
```

Reply:
Done, thanks